lib/collectionspace/tools/date_parser.rb

```diff
@@ -50,27 +52,38 @@ def self.parse(date_string, end_date_string = nil)
             date_string = parse_us_date(date_string)
           end
 
-          parsed_earliest_date = DateTime.parse(date_string)
-          parsed_latest_date = end_date_string ?
-            DateTime.parse(end_date_string) : parsed_earliest_date + 1.send(date_increment)
+          phrase_date_signals = %w[early late cent st nd rd th]
+          phrase_date_signals.each{ |s|
+            if date_string.downcase[s]
+              date_string = ''
+              break
+            end
+          }
```

Comment:
Added this part because DateTime.parse was turning 'Early 20th century' into Mon, 20 Jan 2020 00:00:00 +0000

E.g. when Ruby date parsing tries to be too clever...

I'm sure there's a better way to do this, but couldn't think of it in the moment.

Reply:
First make it work, then make it better (... then being some very far distant future usually for us 😄 )

Reply:
"Then" = when it breaks something noticeable... :-)